samples/testapp/src/commonMain/kotlin/com/android/identity/testapp/ui/CertificateScreen.kt

```diff
@@ -0,0 +1,23 @@
+package com.android.identity.testapp.ui
+
+import androidx.compose.runtime.Composable
+import com.android.identity.appsupport.ui.certificateviewer.CertificateViewer
+import com.android.identity.cbor.Cbor
+import com.android.identity.cbor.CborArray
+import com.android.identity.crypto.X509Cert
+import com.android.identity.crypto.X509CertChain
+import io.ktor.util.decodeBase64Bytes
+
+/**
+ * Populate and compose the certificate chain screen.
+ * Leveraging overloaded composaables from CertificateViewer.kt.
+ *
+ * @param encodedCertificate Base64 encoded Cbor data for the certificate/chain to display.
+ */
+@Composable
+fun CertificateScreen(encodedCertificate: String) {
+    when (val dataItem = Cbor.decode(encodedCertificate.decodeBase64Bytes())) {
```

Comment:
Since you use `ByteArray.toBase64Url(): String` to encode (our own extension function declared in Base64Util.kt) I think it would be proper to use `String.fromBase64Url(): ByteArray` to decode. Mostly for symmetry but also to avoid pulling in Ktor.